### ISSUE TITLE
Conversation StartOnEnter is always true

### ISSUE THREAD

**Issue by MarcusKohnert** [Vonage/vonage-dotnet-sdk#463]
**Describe the bug**
At the moment it is not possible to enable a callee to join a conversation as an attendee. Everyone joins as the moderator.

The differentiator between moderator and attendee is indicated by the property `ConversationAction.StartOnEnter`.
The default value in the object model for that property is `true` as documented [here](https://developer.vonage.com/en/voice/voice-api/ncco-reference#conversation).

However `JsonSerializerSettings` are configured with `DefautValueHandling = DefaultValueHandling.Ignore` [link](https://github.com/Vonage/vonage-dotnet-sdk/blob/2b5fad7864c398b703dd796bfe3f5962a7ebaa48/Vonage/Serialization/VonageSerialization.cs#L13).
This leads to the situation that all boolean properties of an object with a value of `false` are not written to the HTTP response. Since that property isn't written to the HTTP response and the default value for that property in the object model is `true`, it is not possible to produce a `ConversationAction` with a value of `false` for `StartOnEnter`.
Hence no one joins as an attendee and the music on hold is nevery played.

**To Reproduce**
1. Create a ConversationAction with StartOnEnter = false
```cs
var action = new ConversationAction
{
    Name = "some-name",
    StartOnEnter = false,
    MusicOnHoldUrl = new [] { "https://nexmo-community.github.io/ncco-examples/assets/voice_api_audio_streaming.mp3" }
};
```
2. Serialize
```cs
new Ncco(action).ToString();
```

**Expected behavior**
```json
[
    {
        "action": "conversation",
        "name": "some-name",
        "startOnEnter": false,
        "musicOnHoldUrl": [
            "https://nexmo-community.github.io/ncco-examples/assets/voice_api_audio_streaming.mp3"
        ]
    }
]
```

**Actual behavior**
```json
[
    {
        "action": "conversation",
        "name": "some-name",
        "musicOnHoldUrl": [
            "https://nexmo-community.github.io/ncco-examples/assets/voice_api_audio_streaming.mp3"
        ]
    }
]
```

**Additional context**
NuGet package version: 
```xml
<PackageReference Include="Vonage" Version="6.7.0" />
```

**Comment by MarcusKohnert** [comment]
If you want to workaround the issue inherit from ConversationAction and decorate a property with the same name.

```cs
public class CustomConversationAction : ConversationAction
{
    [JsonProperty("startOnEnter")]
    [DefaultValue(true)] // necessary! otherwise the musicOnHold won't be played
    public new bool StartOnEnter { get; set; }
}
```

```cs
var action = new CustomConversationAction
{
	Name = "some-name",
	StartOnEnter = false,
	MusicOnHoldUrl = new [] { "https://nexmo-community.github.io/ncco-examples/assets/voice_api_audio_streaming.mp3" }
};

new Ncco(action).ToString()
```

**Comment by Tr00d** [comment]
Hi @MarcusKohnert,

Thanks for your feedback! I'm going to look into that as soon as possible.
Do you know if the issue was there before? Or has it been introduced recently?

**Comment by MarcusKohnert** [comment]
Hi @Tr00d,

thanks for looking into the issue.
I'm afraid I can't tell you whether that issue has been introduced recently. 
Today was my first try using the conversation object.

**Comment by Tr00d** [comment]
@MarcusKohnert The fix has been released under [v6.7.1](https://github.com/Vonage/vonage-dotnet-sdk/releases/tag/v6.7.1) and should be available for you.
Please update the package, and let me know if it works out

**Comment by MarcusKohnert** [comment]
Works as expected.
